recipes/mutmap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mutmap" %}
+{% set version = "2.0.5" %}
+{% set sha256 = "09c214a55617440acbc03990a7a31de2cbcb85e5756e6ab349687aa928b85a5a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/YuSugihara/MutMap/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+  number: 2
+
+requirements:
+  host:
+    - python >=3.5
+
+  run:
+    - python >=3.5
+    - samtools >=1.7
+    - bcftools >=1.7
+    - bwa
+    - snpeff
+    - trimmomatic
+    - matplotlib
+    - numpy
+    - pandas
+    - seaborn
+
+test:
+  commands:
+    - mutmap -h
+    - mutplot -h
+
+about:
+  home: https://github.com/YuSugihara/MutMap
+  license: GPL
```

Comment:
please add the license_file

Reply:
In your next release, please package the license with the tarballs (you're technically violating the license you've given your own software).